src/TimeService.Models.Db/DbWorkTime.cs

```diff
@@ -1,22 +1,35 @@
-﻿using System;
-using System.ComponentModel.DataAnnotations;
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
 
 namespace LT.DigitalOffice.TimeService.Models.Db
 {
     public class DbWorkTime
     {
-        [Key]
+        public const string TableName = "WorkTimes";
+
```

Comment:
remove empty line

Reply:
Disagree, the table name is not part of the data